doc/manual/modules/performance/ref_built-in-functions-available-in-tuned-profiles.adoc

```diff
@@ -55,3 +55,6 @@ Unpacks a CPU list in the form of `1-3,4` to `1,2,3,4`.
 
 `cpulist_pack`::
 Packs a CPU list in the form of `1,2,3,5` to `1-3,5`.
+
+`intel_recommended_pstate`::
+Returns intel_pstate value based on processor generation
```

Comment:
I think to be more accurate we should say:
Returns recommended intel_pstate CPUFreq driver mode based on processor generation

Reply:
Done